scenariogeneration/xodr/lane.py

```diff
@@ -1109,6 +1118,32 @@ def add_height(self, inner, outer=None, soffset=0):
         self.heights.append(heightdict)
         return self
 
+    def add_lane_material(self, friction, roughness=None, soffset=0, surface=None):
+        """add_lane_material adds a material description entry to the lane
+        Parameters
+        ----------
+            friction (float): friction coefficient
+
+            roughness (float): roughness, for example, for sound and motion systems
+                Default: None
+
+            sOffset (float): s offset of the material
+                Default: 0
+
+            surface (str): surface material code, depending on application
+                Default: None
+
+        """
+        materialdict = {}
```

Comment:
Could create an object, but only used here. Do you think it is better?

Reply:
I'm not really sure. It probably doesn't matter much. I guess it depends on how other similar data is handled, if it's a dict or object, to have cohesion. Another aspect is that it can be easier to find where objects are defined and get an overview of the fields, if they are written as classes, and have corresponding names to the element in the standard. And you can also provide documentation of properties (attributes) in a nice way. But it's a trade off: takes more effort with class and object, but might in some cases make things clearer and modularized.